hack/tools/Makefile

```diff
@@ -18,9 +18,6 @@ HOST_ARCH=$(shell go env GOARCH)
 # Binaries.
 CONTROLLER_GEN  := $(BIN_DIR)/controller-gen
 GOLANGCI_LINT   := $(BIN_DIR)/golangci-lint
-CLIENT_GEN      := $(BIN_DIR)/client-gen
```

Comment:
I think `_ "k8s.io/code-generator"` can be removed from tools.go now, and then run `go mod tidy` to remove that dependency

Reply:
Ah yeah, good call